xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/restore.vm

```diff
@@ -126,7 +129,7 @@
   ## Only propose to restore the batch if it also contains other documents than the current one.
   #if ($batchDeletedDocuments.size() > 1)
     #define($heading)
-      <input name="includeBatch" id="includeBatch" type="checkbox" value="true"/><label for="includeBatch">$services.localization.render('core.restore.includeBatch')</label> ($batchDeletedDocuments.size())
+      <input name="includeBatch" id="includeBatch" type="checkbox" value="true"/><label for="includeBatch">$escapetool.xml($services.localization.render('core.restore.includeBatch'))</label> ($batchDeletedDocuments.size())
```

Comment:
Wouldn't it make sense to move the batch size into the label here, too, if you do this for the children count during deletion?

Reply:
Fixed in https://github.com/xwiki/xwiki-platform/pull/2215/commits/7bc1714558c04e6e883b60fc0ba973eb96ed9446